winton_kafka_streams/processor/serialization/avro.py

```diff
@@ -0,0 +1,68 @@
+from confluent_kafka.avro import CachedSchemaRegistryClient, MessageSerializer
+from confluent_kafka.avro import loads as avro_loads
+
+from ._serde import extract_config_property
+from ._deserializer import Deserializer
+from ._serializer import Serializer
+
+
+class AvroHelper:
+    def __init__(self):
+        self._is_key = False
+        self._schema_registry = None
+        self._serializer = None
+        self._schema = None
+
+    def _set_serializer(self, schema_registry):
+        self._schema_registry = schema_registry
+        self._serializer = MessageSerializer(registry_client=self._schema_registry)
+
+    def configure(self, configs, is_key):
+        self._is_key = is_key
+        schema_registry_url = extract_config_property(configs, is_key, 'AVRO_SCHEMA_REGISTRY')
+        schema = extract_config_property(configs, is_key, 'AVRO_SCHEMA')
+
+        if schema_registry_url is None:
+            raise Exception("Missing Avro Schema Registry Url")
+        else:
+            self._set_serializer(CachedSchemaRegistryClient(url=schema_registry_url))
+
+        if schema:
+            self._schema = avro_loads(schema)
+
+    def serialize(self, topic, data):
```

Comment:
Would it be better both here and in `deserilize` to have a guard that `self._serializer != None`? That way a meaningful error message could be emitted saying that `configure` should be called first.